docs/Metrics.md

```diff
@@ -1,11 +1,48 @@
 # Metrics
 
+1. [Instrumentation (Google Analytics)](#instrumentation-google-analytics)
+2. [A/B Testing](#ab-testing)
+3. [Exchange Rates](#exchange-rates)
+
+
 ## Instrumentation (Google Analytics)
 
-* TODO: add content about GA
-  * convesion tracking
-  * virtual views
-  * react-ga lib
+We load Google Analytics on all of our fundraising web pages, via [index.jsx](https://github.com/mozilla/donate.mozilla.org/blob/master/pages/index.jsx#L16). This is the standard Universal Analytics tracking tag, which gives us default data about page views [code](https://github.com/mozilla/donate.mozilla.org/blob/master/public/js/ga.js).
```

Comment:
I don't know if pointing to a specific line is a good idea here? Might be better to include example instead? Because it often get moves around and then pointing to the wrong line?